PortfolioStyle.css

```diff
@@ -0,0 +1,219 @@
+
```

Comment:
You've got a bit of a mismatch of indentation running through this document, just affects readability slightly.

Reply:
Ugh I hate indentation, thanks, I'll go through